src/createFirebaseInstance.js

```diff
@@ -22,6 +22,8 @@ export default function createFirebaseInstance(firebase, configs, dispatch) {
     firebase.database.enableLogging(configs.enableLogging)
   }
 
+  console.log('Creating firebase instance', firebase)
```

Comment:
@theashguy this is causing CI to fail due to eslint rules. 

If you need this log here might be worth disabling eslint check for this line otherwise you can get rid of it